posthog/api/test/test_error_tracking.py

```diff
@@ -1,65 +1,60 @@
 from posthog.test.base import APIBaseTest
 from posthog.models import Team, ErrorTrackingGroup
+from django.utils.http import urlsafe_base64_encode
+import json
 
 
 class TestErrorTracking(APIBaseTest):
+    def send_request(self, fingerprint, data, endpoint=""):
+        base64_fingerprint = urlsafe_base64_encode(json.dumps(fingerprint).encode("utf-8"))
+        request_method = self.client.patch if endpoint == "" else self.client.post
+        request_method(
+            f"/api/projects/{self.team.id}/error_tracking/{base64_fingerprint}/{endpoint}",
```

Comment:
I don't think the fingerprint should be part of the URL, even more in base64 which is a bit bigger than usual (1/3 bigger)
URLs have a limit of chars and this could break easily (around ~2k chars depending on the browser)
I think the fingerprint should just be the metadata of a group of errors and the group of errors has its id, WDYT?